spk/tvheadend/src/service-setup.sh

```diff
@@ -93,34 +87,34 @@ service_postupgrade ()
 {
     # For backwards compatibility, restore recordings from old package root into var directory
     if [ -d "${SAVE_DIR}" ]; then
-        echo "Restoring recordings into ${HOME_DIR}" >> ${INST_LOG}
-        $CP "${SAVE_DIR}"/. "${HOME_DIR}" >> ${INST_LOG}
+        echo "Restoring recordings into ${HOME_DIR}"
+        $CP "${SAVE_DIR}"/. "${HOME_DIR}"
         $RM "${SAVE_DIR}"
     fi
 
     # Need to enforce correct permissions for recording directories on upgrades
-    echo "Adding ${GROUP} group permissions on recording directories:"  >> ${INST_LOG}
+    echo "Adding ${GROUP} group permissions on recording directories:"
     UPGRADE_CFG_DIR="${SYNOPKG_PKGDEST}/var/dvr/config"
     for file in ${UPGRADE_CFG_DIR}/*
     do
-        DVR_DIR=`grep -e 'storage\":' ${file} | awk -F'"' '{print $4}'`
+        DVR_DIR=$(grep -e 'storage\":' ${file} | awk -F'"' '{print $4}')
         # Exclude directories in @appstore as ACL permissions srew up package installations
         TRUNC_DIR=$(echo "$(realpath ${DVR_DIR})" | awk -F/ '{print "/"$3}')
         if [ "${TRUNC_DIR}" = "/@appstore" ]; then
-            echo "Skip: ${DVR_DIR} (system directory)" >> ${INST_LOG}
-        else
-            echo "Done: ${DVR_DIR}" >> ${INST_LOG}
+            echo "Skip: ${DVR_DIR} (system directory)"
+        elif [ $SYNOPKG_DSM_VERSION_MAJOR -lt 7 ]; then
+            echo "Done: ${DVR_DIR}"
             set_syno_permissions "${DVR_DIR}" "${GROUP}"
         fi
     done
 
     # For backwards compatibility, restore ownership of package system directories
-    echo "Restore '${EFF_USER}' unix permissions on package system directories" >> ${INST_LOG}
+    echo "Restore '${EFF_USER}' unix permissions on package system directories"
     if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 6 ]; then
-        synoacltool -del "${SYNOPKG_PKGDEST}" >> ${INST_LOG} 2>&1
-        chown ${EFF_USER}:root "/var/packages/tvheadend" >> ${INST_LOG} 2>&1
-    else
-        chown ${EFF_USER}:${USER} "/var/packages/tvheadend/target" >> ${INST_LOG} 2>&1
+        synoacltool -del "${SYNOPKG_PKGDEST}"
+        chown ${EFF_USER}:root "/var/packages/tvheadend"
+    elif [ $SYNOPKG_DSM_VERSION_MAJOR == 6 ]; then
```

Comment:
Looking at things, there hasn't been any pre-DSM6 packages released.  All that code can be removed entirely.